src/model/image-environment-factory.ts

```diff
@@ -48,8 +48,8 @@ class ImageEnvironmentFactory {
       { name: 'ANDROID_KEYALIAS_PASS', value: parameters.androidKeyaliasPass },
       { name: 'ANDROID_TARGET_SDK_VERSION', value: parameters.androidTargetSdkVersion },
       { name: 'ANDROID_SDK_MANAGER_PARAMETERS', value: parameters.androidSdkManagerParameters },
-      { name: 'ANDROID_APP_BUNDLE', value: parameters.androidAppBundle },
-      { name: 'EXPORT_AS_GOOGLE_ANDROID_PROJECT', value: parameters.exportAsGoogleAndroidProject },
+      { name: 'ANDROID_APP_BUNDLE', value: parameters.androidAppBundle ? 'true' : 'false' },
+      { name: 'EXPORT_AS_GOOGLE_ANDROID_PROJECT', value: parameters.exportAsGoogleAndroidProject ? 'true' : 'false' },
```

Comment:
Values for these parameters should be determined in `Input` or `Parameters`, not directly in the file that passes them to docker.

Reply:
Not sure what you mean, both are pulled from `parameters` and both are only boolean values, so it looked most clear to me that, these are sent into environment as boolean strings.

Reply:
the image-environment-factory isn't meant to manipulate the values of the parameters. That's what the other two files are for (where the parameters come from).